src/core/main/Globalization/Iso3166Countries.cs

```diff
@@ -284,7 +284,14 @@ public virtual CountryIso3166 Get(string alpha2OrAlpha3OrNumeric)
 
             return countries.FirstOrDefault(x => x.CodeAlpha3 == uppered);
         }
-
+        /// <summary>
+        /// Get All the country list
+        /// </summary>
+        /// <returns>List of CountryIso3166 </returns>
+        public virtual List<CountryIso3166> GetAll()
```

Comment:
It should return an `IReadOnlyList<..>`. Otherwise people might end up changing it.